### PR TITLE
Add changelog for VS 2015 Update 1

### PR DESCRIPTION
**Pull request by dungpa**
Following the tradition of adding changelog in each release, I added entries for VS 2015 Update 1. This is easier to follow than the text from https://github.com/Microsoft/visualfsharp/pulls?q=is%3Apr+is%3Aclosed+milestone%3A%22F%23+4.0+Update+1%22.

I tried to be consistent with the old changelog, let me know if you prefer another format.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,28 @@ Visual F<span>#</span>
 
 All notable changes to this project will be documented in this file.
 
+### [4.0.0] - Visual Studio 2015 Update 1 - 30 November 2015
+
+#### Enhancements
+* Perf: 'for i in expr do body' optimization [#219](https://github.com/Microsoft/visualfsharp/pull/219)
```
Comment (enricosada) [inline CHANGELOG.md]: use backtick, to format as `for i in expr do body`

Comment (dungpa) [inline CHANGELOG.md]: Ok, done.

**Comment by KevinRansom** [comment]
Thank you this is great.

**Comment by NumberByColors** [comment]
Thanks, @dungpa! Was half-through putting this together. Beat me to it. :)

**Comment by dungpa** [comment]
My pleasure :)

**Pull request merged**